Lib/fontTools/subset/__init__.py

```diff
@@ -2505,7 +2490,7 @@ class UnknownOptionError(OptionError): pass
 
 	# spaces in tag names (e.g. "SVG ", "cvt ") are stripped by the argument parser
 	_drop_tables_default = ['BASE', 'JSTF', 'DSIG', 'EBDT', 'EBLC',
-				'EBSC', 'SVG', 'PCLT', 'LTSH']
```

Comment:
I don't think you necessarily have to change the drop default. Maybe that should be in it's own PR since it's not required to support subsetting?

Reply:
normally when we add support for a new table, we also remove it from the drop-tables list, which is the list of tables for which the subsetter does not know yet how to subset so it simply drops them by default.

What do you mean "it's not required to support subsetting"?

Reply:
If the convention is if it works don't drop it then it makes sense to remove it. I was interpreting it in a somewhat different light, more like tables we recommend be used. In retrospect several other things would drop by default if that was true so ... ignore me :D